src/components/AccountDetails/index.tsx

```diff
@@ -214,11 +215,14 @@ export default function AccountDetails({
 }: AccountDetailsProps) {
   const { chainId, account } = useActiveWeb3React()
   const dispatch = useDispatch<AppDispatch>()
+  const { t } = useTranslation()
 
   const clearAllTransactionsCallback = useCallback(() => {
     if (chainId) dispatch(clearAllTransactions({ chainId }))
   }, [dispatch, chainId])
 
+  const addressLinkText = t('viewOnBlockExplorer')
```

Comment:
this is a bit odd, having a variable with the translation. Why not use the translation directly?

Reply:
Ok, it's used twice.